src/rgw/rgw_rest_role.h

```diff
@@ -7,6 +7,7 @@
 
 #include "rgw_role.h"
 #include "rgw_rest.h"
+#include "rgw_rest_iam.h"
```

Comment:
doesn't look like this iam include is needed in the header?

Reply:
removed